packages/visual-editor/src/components/Draggable/DraggableComponent.tsx

```diff
@@ -63,7 +65,9 @@ export const DraggableComponent = ({
           onClick={onClick}>
           {!isSelected ? (
             <div
-              className={classNames(styles.overlay, {
+              className={classNames({
+                [styles.overlayContainer]: isContainer,
+                [styles.overlay]: !isContainer,
```

Comment:
I just had an idea: you could make the `.overlay` styles always present here (static), and apply the `.overlayContainer` styles conditionally based on your `isContainer` boolean.. Then the actual css within `.overlayContainer` only needs to be the styles you want to change/override from `.overlay` (in this case only the `top` style is different and all the other styles are duplicates (which could just be inherited instead of duplicated)

```suggestion
              className={classNames(styles.overlay, {
                [styles.overlayContainer]: isContainer,
```